docs/template/public/less/_variables.less

```diff
@@ -22,4 +22,11 @@
 @dark-blue: #27394f;
 @side-nav-link: #dbe3eb;
 @top-nav-hover: #0376C6;
-@btn-type-color: #0b0080;
\ No newline at end of file
+@btn-type-color: #0b0080;
+@kss-green: #04ce26;
```

Comment:
@saadmhybris Please move variables that are used exclusively for the page styling out of the normal variables file. into the kss.less. If you want to use pre-existing variables of the framework in the kss.less please make a import of the less file.